cmd/internal/su.go

```diff
@@ -47,3 +41,64 @@ func AutoSu() {
 	}
 	os.Exit(stat.ExitCode())
 }
+
+func trySudo() (path string, arg []string) {
+	pathSudo, err := exec.LookPath("sudo")
+	if err != nil {
+		// fallback
+		var possibleSudoPath = []string{
+			"/usr/bin/sudo", "/usr/sbin/sudo",
```

Comment:
Please don't hardcode the absolute path of `sudo`, since some Linux distros don't follow FHS (such as NixOS).

Reply:
Thank you for bringing this up. I'm not familiar with NixOS's file system hierarchy. Could you provide guidance on the proper way to handle this for NixOS compatibility? I'll make the necessary adjustments in upcoming commits.

Reply:
using `exec.LookPath` is enough

Reply:
@woshikedayaa Just use `exec.LookPath` to check the existence of `pkexec` is sufficient.